sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -96,7 +110,7 @@ export class EducationProgramController {
     @Body() payload: EducationProgramDto,
     @UserToken() userToken: IInstitutionUserToken,
   ): Promise<number> {
-    const newProgram = await this.saveProgram(userToken, payload);
+    const newProgram = await this.saveProgram(userToken, payload, undefined);
```

Comment:
Do we need to provide undefined here? I believe that we can just invoke the method as below, right?
```javascript
this.saveProgram(userToken, payload,)
```